src/pages/HomePage/HomePage.js

```diff
@@ -1,71 +1,58 @@
-          <div class="category">
-            <img class="category-img img-responsive" src={categoryImage5} alt="bodycare-category-img"/>
-            <div className='category-text'> <h5 class="category-title-text">Body care</h5><p className='category-text-secondary'>We bring bodycare products at best prices.</p></div>
-          </div>
-        </Link>
-        <Link to="/productlistingpage" className='links' >
-          <div class="category">
-            <img class="category-img img-responsive" src={categoryImage2} alt="menstrual-hygiene-category-img"/>
-            <div className='category-text'> <h5 class="category-title-text">Menstrual Hygiene</h5><p className='category-text-secondary'>Buy Menstrual Hygiene care products at most affordable prices now.</p></div>
-          </div>
-        </Link>
-      </div>
+     <Categories/>
       <h5 className='sub-heading'>Offers you won't miss!</h5>
       <div className='offers-section'>
         <img className="offer-img" src={offerImage1} alt="offer-img-1"/>
         <img className="offer-img" src={offerImage2} alt="offer-img-2"/>
       </div>
-      <h3 className='sub-heading'>Featured Brands</h3>
+      <h3 className='sub-heading'>Featured Top Brands</h3>
       <div className='brands-section'>
-        <img className='brand-img' src={brandImage1} alt="brand-img"/>
-        <img className='brand-img' src={brandImage2} alt="brand-img"/>
-        <img className='brand-img' src={brandImage4} alt="brand-img"/>
-        <img className='brand-img' src={brandImage5} alt="brand-img"/>
+        <Link to="/productlistingpage"><img className='brand-img' src={brandImage1} alt="brand-img" onClick={(e) => dispatch({ type: 'SET_BRAND', payload: "L'Oreal" }, dispatch({ type: 'SET_SUB_CATEGORY', payload: null }))}/></Link>
+        <Link to="/productlistingpage"><img className='brand-img' src={brandImage2} alt="brand-img" onClick={(e) => dispatch({ type: 'SET_BRAND', payload: "Lakme" }, dispatch({ type: 'SET_SUB_CATEGORY', payload: null }))} /></Link>
+        <Link to="/productlistingpage"><img className='brand-img' src={brandImage4} alt="brand-img" onClick={(e) => dispatch({ type: 'SET_BRAND', payload: "Maybelline" }, dispatch({ type: 'SET_SUB_CATEGORY', payload: null }))} /></Link>
+        <Link to="/productlistingpage"><img className='brand-img' src={brandImage7} alt="brand-img" onClick={(e) => dispatch({ type: 'SET_BRAND', payload: "Whisper" }, dispatch({ type: 'SET_SUB_CATEGORY', payload: null }))}/></Link>
+        <Link to="/productlistingpage"><img className='brand-img' src={brandImage5} alt="brand-img" onClick={(e) => dispatch({ type: 'SET_BRAND', payload: "Mamaearth" }, dispatch({ type: 'SET_SUB_CATEGORY', payload: null }))} /></Link>
+        <Link to="/productlistingpage"><img className='brand-img' src={brandImage6} alt="brand-img" onClick={(e) => dispatch({ type: 'SET_BRAND', payload: "Wow" }, dispatch({ type: 'SET_SUB_CATEGORY', payload: null }))}/></Link>
+      
```

Comment:
You can make an array of FeaturedBrands having the imageUrl and respective payload values which you can map over and add the links here